website/docs/r/mssql_elasticpool.html.markdown

```diff
@@ -92,14 +92,14 @@ The following arguments are supported:
 
 ---
 
+* `zone_redundant` - (Optional) Whether or not this elastic pool is zone redundant. `tier` needs to be `Premium` for `DTU` based  or `BusinessCritical` for `vCore` based `sku`. Defaults to `false`.
```

Comment:
@katbyte  should we leave the Defaults to false here? As its the API behavior anyways its still may a fair hint?

Reply:
I would leave it as yes that is what the api does by default.